pyne/CMakeLists.txt

```diff
@@ -6,14 +6,27 @@ set(SRC_FILES pyne.cpp)
 # Public headers
 set(PUB_HEADERS pyne.h)
 
-# Linked libraries
-set(LINK_LIBS        ${DAG_LINK_LIBS})
-set(LINK_LIBS_STATIC ${DAG_LINK_LIBS})
+# Optimization flags: always use -O0
+set(CMAKE_CXX_FLAGS_RELEASE "-O0")
```

Comment:
It's an intel thing. If you try to compile pyne.cpp with intel and anything other than `-O0`, it compiles for a few minutes and then crashes my computer due to it running out of memory (I have 16 GB).